CIP-xxxx/README.md

```diff
@@ -0,0 +1,145 @@
+
+### ED25519-BIP32 Private Key
+
+This key is defined in [ED25519-BIP32].
+
+This is encoded as a byte string of size 32 bytes.
+
+#### [CDDL]
+
+```cddl
+ed25519_private_key = #6.32771(bstr .size 32)
+```
+
+Data for the key inside the byte string is encoded in [network byte order].
+
+### ED25519-BIP32 Extended Private Key
+
+This key is defined in [ED25519-BIP32].
+
+This is encoded as a byte string of size 64 bytes.
+
+#### [CDDL]
+
+```cddl
+ed25519_extended_private_key = #6.32772(bstr .size 64)
+```
+
+Data for the key inside the byte string is encoded in [network byte order].
+
+### ED25519-BIP32 Public Key
+
+This key is defined in [ED25519-BIP32].
+
+This is encoded as a byte string of size 32 bytes.
+
+#### [CDDL]
+
+```cddl
+ed25519_public_key = #6.32773(bstr .size 32)
+```
+
+Data for the key inside the byte string is encoded in [network byte order].
+
+### ED25519-BIP32 Signature
+
+[ED25519-BIP32] defines how signatures can be generated on data from private keys.
+These signatures are defined to be 64 bytes long.
+
+Signatures are encoded as a byte string of size 64 bytes.
+
+#### [CDDL]
+
+```cddl
+ed25519_bip32_signature = #6.32774(bstr .size 64)
+```
+
+Data for the signature inside the byte string is encoded in [network byte order].
+
+## Rationale: how does this CIP achieve its goals?
+
+By defining concrete CBOR tags,  it is possible for metadata to unambiguously mark the kind of data encoded.
+This is conformant with the intent of Tags in [CBOR], and aligns with [CIP-CBOR-TAGS].
+
+An official published spec is required to register these Tags with [IANA][IANA CBOR Tag Registry].
+This document also serves that purpose.
+
+## Path to Active
+
+### Acceptance Criteria
+
+* At least 1 Metadata CIP uses at least 1 of the tags defined in this CIP.
+* IANA register the Tags as defined.
+
+### Implementation Plan
+
+* Some of these Tags will be used by Project Catalyst for CBOR data structure definitions.
+* Project Catalyst will also make the application to [IANA][IANA CBOR Tag Registry] to register the Tags.
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0]
+
+Code samples and reference material are licensed under [Apache 2.0]
+
+[CC-BY-4.0]: https://creativecommons.org/licenses/by/4.0/legalcode
+[Apache 2.0]: https://www.apache.org/licenses/LICENSE-2.0.html
+[CBOR]: https://www.rfc-editor.org/rfc/rfc8949.html
+[CDDL]: https://www.rfc-editor.org/rfc/rfc8610
+[CIP-0003]: https://cips.cardano.org/cip/CIP-0003
+[BIP32]: https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki
+[IANA CBOR Tag Registry]: https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml
+[network byte order]: https://datatracker.ietf.org/doc/html/rfc1700
+[ED25519-BIP32]: https://github.com/input-output-hk/adrestia/raw/bdf00e4e7791d610d273d227be877bc6dd0dbcfb/user-guide/static/Ed25519_BIP.pdf
+[CPS-CBOR-TAGS]: https://github.com/cardano-foundation/CIPs/pull/751
+[CIP-CBOR-TAGS]: https://github.com/cardano-foundation/CIPs/pull/752
```

Comment:
@rphair can you provide context on recommendation per the new update to accept a **references** section in the standard template

Reply:
@Crypto2099 yes... @stevenj I think our readers would benefit for creating markup for these, all given a heading of `## References` (and moving it before the _Copyright_) as recently endorsed here (see _optional sections_): https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#structure

Reply:
Done